src/answers-search-bar.js

```diff
@@ -260,20 +209,9 @@ class AnswersSearchBar {
 
     this.renderer.init(parsedConfig.templateBundle, this._getInitLocale());
     this._onReady();
-    if (!this.components.getActiveComponent(SearchComponent.type)) {
-      this._initQueryUpdateListener(parsedConfig.search);
-    }
     this._searchOnLoad();
```

Comment:
I think we can get rid of searchOnLoad? The SearchComponent's searchAfterAnswersOnReady() guards it from doing anything when there is a redirectUrl.

Reply:
Removed! Thanks for pointing that out.